shared/data/fingerprint-protection.js

```diff
@@ -68,6 +68,18 @@
                 'targetValue': undefined
             }
         },
+        'useragent': {
+            'userAgent': {
+                'object': 'navigator',
+                'origValue': navigator.userAgent,
+                'targetValue': `"${ua}"` // Defined in chrome-events.es6.js and injected as a variable
+            },
+            'appVersion': {
+                'object': 'navigator',
+                'origValue': navigator.appVersion,
+                'targetValue': /Firefox/i.test(navigator.userAgent) ? `"${navigator.appVersion}"` : `"${ua.replace('Mozilla/', '')}"` 
```

Comment:
hm, I'm not sure if I understand what's going on here and why 🤔 Might be worth extracting to a seprate method with a comment?

Reply:
Thanks for noticing this - definitely it's confusing (even going back over now a few days after I wrote it I had to remind myself of a couple of things). I expanded it into a function with liberal comments.